tests/data/dhcp.leases.custom.dns

```diff
@@ -0,0 +1,56 @@
+lease {
```

Comment:
Is this a typical example?  

The same lease is repeated three times. Is this is expected. When you are searching for `option unknown-245` should you look for the first or last? Does it matter?  Should you take the interface into account, e.g. interface "eth0" when parsing?

Reply:
This is the same file as used for the other unit tests (except for the last entry, which has a custom DNS server: 8.8.8.8). I would say it is typical, looks very similar to the one on my VM. According to man, the file acts as a log and the last entry should always be used.

Actually, I need to update the address in option 245 to a dummy address, to ensure the last one is being picked (with the previous code this was ensured by using 'invalid' and 'expired' in domain-name-servers of the first 2 entries.

Reply:
I just updated the test file -- added dummy wireserver addresses to the first 2 entries to ensure the test is pickling up the last one (third one).